chatwoot/chatwoot
# feat: Adds the ability to search emojis and render

# Pull Request Template

## Type of change


## How Has This Been Tested?
https://www.loom.com/share/a2798dd9fbd44320a495353dbc798704

**Test cases**
1. Search emojis from the emoji box search.
2. See all places where the emoji box Is used. (widget, reply box, expanded reply box)


## Checklist:
